tools/cloud_run/event_driven_bq_ingest/src/main/java/com/example/cloudrun/PubSubMessageProperties.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.cloudrun;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class PubSubMessageProperties {
```

Comment:
If you're going to have a class like this, you should have it also store bq job notifications which come in through pubsub.